scripts/build.sh

```diff
@@ -2,25 +2,52 @@
 
```

Comment:
We should update the documentation on readme.md to say that #48 is fixed if you use MINGW.